docs/intents/ref/StartCall.md

```diff
@@ -38,9 +38,11 @@ fdc3.raiseIntent('StartCall', contact)
 ## See Also
 
 Context
-- [Contact](../../context/ref/Contact)
-- [ContactList](../../context/ref/ContactList)
+
+* [Contact](../../context/ref/Contact)
```

Comment:
The linter picks whichever it first sees in the file and is happy with either as long as they are consistent. These are flagged as the files use a mix of - and *

Reply:
I can go and change them all everywhere to the other way, but don't think it wholly necessary as I find the stars as readable. I'd just prefer we are consistent throughout

Reply:
I just don't see the point of consistency when it comes to bullets. It is the kind of arbitrary rule that is just pedantic without any reason. So long as the _same_ bulleted list doesn't use mixed dashes and stars, it should be fine.

Flexibility is good - e.g. in the case where you have bolded items in the bulleted list, it is often better to use dashes, so that you can more easily distinguish the bullets from the bold text. 🤷‍♂️ 

Anyway, this is minor. If others are happy to impose this kind of dictatorship on our markdown files, I can live with it.

Reply:
I do most of the markdown editing in our docs currently and think consistency is good and find that the linter helps me to catch inadvertant mark-up errors (which sometimes slip through review and cause rendering errors in the site)  which I why I apply it. 

However, I also prefer the dash style (dut don't really have a problem with either), so I'll go back and change every bullet to a dash. What happened here is that the first bullets used were stats then the style switched to dashes so that's what ended up being used.

Reply:
Okay - wasn't meaning to create extra work for you @kriswest! It's okay to merge like this, my biggest concerns where the headings.

Reply:
All done and standardized on the hyphen style! (it is more readable in those bold cases + I like it better + I'd missed a file files that had already gone to hyphens)